src/routes/organization/[organizationId]/+page.server.ts

```diff
@@ -2,7 +2,6 @@ import {
   checkSessionAuth,
   getOrganizationInviteDetails,
   getOrganizationMemberDetails,
-  isUserGlobalAdmin,
```

Comment:
why was this removed?

Reply:
It wasn't being used. For some reason on this repo that is not a linting error.